web/src/components/HeaderActionItemDropDown.tsx

```diff
@@ -37,12 +38,18 @@ const HeaderActionItemDropDown = ({ iconSrc, text, children }: HeaderActionItemD
   const wrapperRef = useRef(null)
   useOnClickOutside(wrapperRef, closeDropDown)
 
+  const id = spacesToDashes(text)
+
   return (
     <div ref={wrapperRef}>
-      <Tooltip text={text} flow='down' mediumViewportFlow='left'>
-        <Button label={text} onClick={toggleDropDown}>
-          <StyledIcon src={iconSrc} />
-        </Button>
+      <Tooltip
+        id={id}
+        container={
+          <Button label={text} onClick={toggleDropDown} id={id}>
+            <StyledIcon src={iconSrc} />
+          </Button>
+        }>
+        {text}
       </Tooltip>
```

Comment:
:upside_down_face: In general I think this component would be easier to use and the props would be clearer if `container` and `children` was reversed imo, i.e. you would pass components as children the same way as before.
Then the `Tooltip` component could have a required prop `text` as before and an optional property `container`, with the latter only necessary in few cases when the default tooltip layout is not enough.

This is only a personal preference though since I think that using these `container` props make it kinda hard to read/understand. Since you already put a lot of effort in this, this is completely up to you (or you can just tell me to do it myself, then I'll push a commit to change this).

Reply:
Hmmm, good point, it's probably clearer in that direction, I'll update it.

(Love the comment that I can also tell you to ***** do it yourself 😄 )